lib/index.js

```diff
@@ -21,15 +21,21 @@
  * @copyright Copyright (c) 2017 Tozny, LLC (https://tozny.com)
  */
 
-import { default as ClientInterface } from 'e3db-client-interface'
+import { Config, types } from 'e3db-client-interface'
 import SodiumCrypto from './SodiumCrypto'
+import helpers from './helpers'
+import Client from './client'
 
-export { types } from 'e3db-client-interface'
-export { default as helpers } from './helpers'
+export { Client, Config, types, helpers, SodiumCrypto }
 
-const crypto = new SodiumCrypto()
-/* eslint-disable-next-line new-cap */
-export const Config = ClientInterface.Config
-/* eslint-disable-next-line new-cap */
-export const Client = ClientInterface.Client(crypto)
-Client.crypto = crypto
+const Tozny = {
+  types,
+  Storage: {
+    Client,
+    Config
+  },
+  helpers,
```

Comment:
whimsy: what about `help` , leading to future calls like `tozny.help.b64URLEncode()` in our customers code?